### PR TITLE
[Site] Add alt and title attributes for the images

### PR DESCRIPTION
**Pull request by pastukhVitalii**

### CODE REVIEW

File: src/hooks/queries/useHomeGalleryAssets.js [hunk 1]
```diff
@@ -91,21 +91,33 @@ const useHomeGalleryAssets = () => {
     if (img.src.includes('1.jpg')) {
       img.width = LARGE_PICTURE.WIDTH;
       img.height = LARGE_PICTURE.HEIGHT;
+      img.alt = 'workspace';
```
Comment (lazio) [inline src/hooks/queries/useHomeGalleryAssets.js:94]: can you write relevant alts and titles?

File: src/scenes/BlogPage/components/Previews/components/List/List.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ const STEP_VALUE = 6;
 const List = ({ items }) => {
   const [numberOfRendered, setNumberOfRendered] = useState(STEP_VALUE);
   const postsToRender = [];
-
+  console.log(items);
```
Comment (lazio) [inline src/scenes/BlogPage/components/Previews/components/List/List.js:13]: please remove

File: src/scenes/HomePage/components/Hero/Hero.module.scss [hunk 3]
```diff
@@ -149,35 +149,3 @@
   }
 }
 
-@keyframes gradient-x {
```
Comment (lazio) [inline src/scenes/HomePage/components/Hero/Hero.module.scss:152]: don't we need this code? 
anything changed after you removed it?

File: src/scenes/Projects/components/List/List.js [hunk 4]
```diff
@@ -31,6 +31,8 @@ const List = ({
       preview: preview1,
       tags: 'UX, UI, Illustrations, Icons',
       title: 'Reinvented bedding cutting price for 50%',
+      alt: 'Primary Smart Bedding Website',
```
Comment (lazio) [inline src/scenes/Projects/components/List/List.js:34]: good alt text 👍

**Review by lazio — CHANGES_REQUESTED** [review]

File: src/hooks/queries/useHomeGalleryAssets.js (hunk 1 above)
Comment (pastukhVitalii) [reply src/hooks/queries/useHomeGalleryAssets.js:94]: 1) Pleasant working atmosphere
2) Cozy workplace

File: src/scenes/HomePage/components/Hero/Hero.module.scss (hunk 3 above)
Comment (pastukhVitalii) [reply src/scenes/HomePage/components/Hero/Hero.module.scss:152]: Not needed,
nothing changed

**Review by lazio — APPROVED** [review]

**Pull request merged**